### PR TITLE
[WIP] Update antigen completion

### PR DESCRIPTION
**Pull request by knakayama**
As discussed on [#26](https://github.com/zsh-users/antigen/issues/26), currently antigen completion does not include subcommand arguments.
So I tried to update it. But since first writing the zsh completion, it may be wrong, and aruguments may not be all included (I have not seen the whole source code yet). So that, I wip this PR. Please review this PR, so I fix it.

### CODE REVIEW

File: antigen.zsh [hunk 1]
```diff
@@ -737,20 +737,55 @@ antigen () {
...
+    'apply:Load all bundle completions'
+    'snapshot:Create a snapshot of all the active clones'
+    'restore:Restore the bundles state as specified in the snapshot'
+    'selfupdate:Update antigen itself'
+    'help:Print help message'
+  )
+
+  __bundle() {
+    _arguments \
+      '--loc=[Path to the location <path-to/location>]' \
```
Comment (desyncr) [inline antigen.zsh]: Bundle is missing 'type' (--btype) argument; which values can be either "theme" or "plugin". See [here](https://github.com/zsh-users/antigen/blob/master/antigen.zsh#L29) and here [here](https://github.com/zsh-users/antigen/blob/master/antigen.zsh#L62).

**Comment by desyncr** [comment]
Thanks for the work. So far I just saw I little thing that I commented above other than that :+1:

**Comment by knakayama** [comment]
@desyncr 
Thanks. I add `--btype` completion.

**Comment by desyncr** [comment]
Works smoothly :+1: I don't think there is any missing argument so it's all OK. Thanks again for the work!

**Comment by desyncr** [comment]
Merged at https://github.com/zsh-users/antigen/pull/155. Thanks @knakayama

**Pull request closed without merging**